app/admin.js

```diff
@@ -108,6 +106,15 @@ setVerificationCommand.addArgument(
   }
 );
 
+const recoveryInfoCommand = subparsers.addParser('recoveryInfo', { addHelp: true });
```

Comment:
Here we are adding a new command. This command is meant to run online, and it is the first step in a recovery flow. This command queries the Database to find the necessary info for a wallet recovery.